extensions/blocks/podcast-episodes/attributes.js

```diff
@@ -0,0 +1,12 @@
+const urlValidator = url => ! url || url.startsWith( 'http' );
```

Comment:
You could look at `isUrl()` from `@wordpress/url`: https://github.com/WordPress/gutenberg/blob/03e3c05eca3ee66b99655e79d2939dafe27a46e3/packages/url/src/is-url.js#L16